### DISCUSSION TITLE
Adjusting Hardness Levels

### DISCUSSION THREAD

**Discussion by steveman1123** [PintaProject/Pinta#980] [Ideas]
Can there be a setting next to "Brush Width" that adjusts hardness level? It'd be superb to able to erase/clone/paint with a softer brush than what the smooth setting can apply.

If there's a setting that I'm unaware of, I'd be happy to use that too.

**Comment by cameronwhite** [comment]
I think this sounds the same as issue #751?

**Reply by steveman1123** [reply]
Exactly the same issue. I searched for "hardness" before posting, but there were no issues that came up

**Comment by steveman1123** [comment]
Dupe of #751
